src/main/java/titanicsend/app/autopilot/utils/TEMixerUtils.java

```
@@ -31,6 +31,10 @@ public static void turnDownAllChannels(LX lx, boolean onlyAffectPhraseChannels)
 
     public static void setFaderTo(LX lx, TEChannelName name, double faderLevel) {
         try {
+        	// Don't crash
+        	if (lx.engine.mixer.channels.size() <= name.getIndex())
+        		return;
```

Probably want to log a NOISY error here... as we're definitely in a bad state if this is the case and wouldn't want to fail silently.

---

I'm checking for null on the return.  The scenario is that someone turned on autopilot and they're aren't enough channels.  The autopilot wants particular channel indexes.  Yeah it's a "problem", but only soft error.  Not hard error, because don't crash the art when some curious VJ clicks autopilot=on to "see what is does" and there's only 2 channels...